packages/expandables/src/expandables.less

```diff
@@ -34,9 +34,9 @@
 
 // Import external dependencies
 
-@import (less) '../../cf-core/src/cf-core.less';
-@import (less) '../../cf-buttons/src/cf-buttons.less';
-@import (less) '../../cf-icons/src/cf-icons.less';
+@import (less) '../../core/src/core.less';
+@import (less) '../../buttons/src/buttons.less';
```

Comment:
Can you think of any scenario where it becomes a pain in a code search to look for `buttons.less` since it's more generic?

Reply:
I was thinking the same thing but it felt weird leaving `cf-` in filenames. Ideally we'd just import the node module name a la `@import '@cfpb/buttons';` and gulp would get the module's entry point based on its package.json `less` or [`style` attribute](https://jaketrent.com/post/package-json-style-attribute/). In fact, I'm surprised we never configured CF to operate like that.

Reply:
Sure I'm fine with `cf-` or `cfpb-`… or `bcfp-` KIDDING